core/commands/refs.go

```diff
@@ -52,7 +52,9 @@ with the following format:
 
   <link base58 hash>
 
-NOTE: List all references recursively by using the flag '-r'.
+List all references recursively by using the flag '-r'.
+
+NOTE: Kubo will try to fetch the blocks of the passed path if they can't be found in the local store.
```

Comment:
```suggestion
NOTE: Like most other commands, Kubo will try to fetch the blocks of the passed path if they can't be found in the local store if it is running in online mode.
```
This resolve the issue of having an inconsistent message on this command, if you do this and it's helping you with your docs then it's mergeable :slightly_smiling_face:.